assets/js/no-in-person-locations-display.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
```

Comment:
Can we turn on the "automatic" runtime for the Babel React preset, so that we don't need to explicitly import React like this?

Like we have in the IdP: https://github.com/18F/identity-idp/blob/f4bcac3/babel.config.js#L13-L18